poom/main_menu.py

```diff
@@ -0,0 +1,267 @@
+from abc import ABC, abstractmethod
+from os import getcwd
+from pathlib import Path
+
+import pygame
+import pygame_gui
+
+# "#2f353b"
+pygame.init()
+size = width, height = 800, 600
+screen = pygame.display.set_mode(size)
+pygame.display.set_caption("Greeting")
+root = Path(getcwd())
+background = pygame.transform.scale(
+    pygame.image.load(root / "assets" / "back.png").convert_alpha(), (width, height)
+)
+running = True
+clock = pygame.time.Clock()
+manager = pygame_gui.UIManager(size, root / "assets" / "style.json")
+animation = pygame.sprite.GroupSingle()
+
+
+class Zombie(pygame.sprite.Sprite):
+    def __init__(self):
+        super().__init__(animation)
+        self.width, self.height = 111, 162
+        self.images = [
```

Comment:
Maybe create a new class for animation?
It will separate different animations, like walking and rotating.
Also, animations changes can be implemented using the State pattern. (but in case with 2 animations it might be overengineering)

I see usage of it like that:
```python
# Inside __init__()
self._walking_animation = Animation(path_to_walking_anim)
self._rotation_animation = Animation(path_to_rotation_anim)

# Inside update(self, dt: float) -> None
if self.rect.left <= 0 or self.rect.right >=:
   self._rotation_animation.play(dt)
   # Changing direction and pushing away from the wall
 else:
    self._walking_animation.play(dt)
    # Moving...
```